formfactory/models.py

```diff
@@ -183,3 +184,60 @@ class Meta:
 
     def __unicode__(self):
         return self.title
+
+
+class Wizard(models.Model):
+    """
+    Wizard model that groups forms together.
+    """
+    title = models.CharField(
+        max_length=30, help_text=_("A short descriptive title.")
+    )
+    slug = models.SlugField(
+        max_length=40, db_index=True, unique=True
+    )
+    forms = models.ManyToManyField(Form, through="FormOrderThrough")
+    actions = models.ManyToManyField(Action, through="WizardActionThrough")
+
+    @cached_property
```

Comment:
If this property is not absolutely necessary rather don't query in a property. This means that an additional query will be made every time this object is instantiated. Also note cached_properties only cache for the length an object is instantiated while may be a very short time.